### PR TITLE
Update StandardCodec documentation with double alignment

### PR DESCRIPTION
**Pull request by robert-ancell**

### CODE REVIEW

File: packages/flutter/lib/src/services/message_codecs.dart [hunk 1]
```diff
@@ -243,7 +243,8 @@ class StandardMessageCodec implements MessageCodec<dynamic> {
   // * Larger integers are encoded using 8 bytes two's complement
   //   representation.
   // * doubles are encoded using the IEEE 754 64-bit double-precision binary
-  //   format.
+  //   format. Zero bytes are added before a double to align the value to a 64
```
Comment (goderbauer) [inline packages/flutter/lib/src/services/message_codecs.dart:246]: Not sure I fully understand, a IEEE 754 64-bit double-precision value is always 64bit long and doesn't need any alignment?

Comment (robert-ancell) [reply packages/flutter/lib/src/services/message_codecs.dart:246]: Each double value has padding before it so its starts on a 64 bit boundary from the start of the message. The other aligned elements in the standard codec are Int32List, Int64List and Float64List which have "the smallest number of zero bytes needed to align the position in the full message with a multiple of the number of bytes per element".

Both comments aren't super clear to me, but I couldn't think of anything clearer at the time.

Comment (goderbauer) [reply packages/flutter/lib/src/services/message_codecs.dart:246]: Maybe we should say instead:

Zero bytes are added before the encoded double value to align it to a 64 bit boundary in the full message.

**Review by goderbauer — APPROVED** [review]
LGTM

**Pull request merged**